modules/flowable-engine-common/src/test/java/org/flowable/common/engine/impl/el/JsonNodeELResolverTest.java

```diff
@@ -0,0 +1,85 @@
+package org.flowable.common.engine.impl.el;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.flowable.common.engine.impl.javax.el.ELContext;
+import org.flowable.common.engine.impl.javax.el.ELResolver;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Prathamesh Mane
+ */
+class JsonNodeELResolverTest {
+    private final ELResolver resolver = new JsonNodeELResolver();
+    private final ELContext context = new FlowableElContext(resolver, null);
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @Test
+    void integerValueShouldBeIntNode() {
+        ObjectNode node = mapper.createObjectNode();
+        resolver.setValue(context, node, "age", 11);
+        assertTrue(node.get("age").isInt());
```

Comment:
The project's standard testing framework is assertj; please see https://assertj.github.io/doc/

Reply:
Done